apps/web/src/pages/update-submission/confirmation.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { Link } from '@components';
+
+const Confirmation: React.FC = () => {
+  const { push, query, isReady } = useRouter();
+  const id: string = query.id as string;
+
+  useEffect(() => {
+    // redirect to the home page if the id is not present
+    if (isReady && !id) {
+      push('/');
+    }
+  });
```

Comment:
how come the missing dependency as opposed to `id`?

Reply:
I can clear this out. I copied this template from the confirmation page for the main form. An ID is not needed here.

Reply:
ah ok, was just curious if it was on purpose